models.yml

```diff
@@ -3874,10 +3874,18 @@ projector:
     type: color
     default: "#317796"
     restriction_mode: A
+  chyron_background_color_2:
+    type: color
+    default: "#134768"
```

Comment:
Where does this default come from?

Reply:
Same colour as the global headbar in default theme

Reply:
Okay, I think it should be fine then. Nice.